do/zones.go

```diff
@@ -32,15 +32,15 @@ func newZones(client *godo.Client, region string) cloudprovider.Zones {
 	return zones{client, region}
 }
 
-// GetZone returns a cloudprovider.Zone for the currently running node.
-// GetZone will only fill the Region field of cloudprovider.Zone for DO.
+// GetZone returns a cloudprovider.Zone from the region of z. GetZone only sets
+// the Region field of the returned cloudprovider.Zone.
```

Comment:
But that's not what it does; it gets the region field of the receiver, `z`. That's only the region of the currently running program because of how the receiver is currently constructed. I think this is correct as is.

Reply:
I take your point, and added some doc that I hope will satisfy your concern.